smk/QC_2.smk

```diff
@@ -161,13 +161,15 @@ if sum(flags) == 0:
 else:
     print('ERROR in ', config_path, ': TAXA_profiler variable is not correct. "TAXA_profiler" variable should be metaphlan, motus_rel or motus_raw, or combinations thereof.')
 
-# TODO: Read this from yaml file and remove hard-coding
```

Comment:
In the same spirit as this comment that is now removed, let us raise an Exception complaining that versions are missing, only for the profilers defined via `taxonomies`.